sdk/monitor/monitor-opentelemetry/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
-## 1.7.1 ()
+## 1.7.1 (2024-09-13)
```

Comment:
Let's update exporter changelog as well, we should have at least dependency updates in that package

Reply:
I added the release date; the changes for other exporter PRs were already in the changelog